src/deadline/client/api/_attachment.py

```diff
@@ -0,0 +1,82 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I didn't notice, but this is an API of `deadline.client`!

WDYT if this was `deadline.job_attachments.api` ? The long term goal is to have job_attachments as a standalone library, so having the API separate would go some way. Its structured already in this fashion in my PR.

Reply:
https://github.com/aws-deadline/deadline-cloud/pull/453/files#diff-db751fc158b122869a89a089141f40449e2fc8402e746566ca5cd6e3a2500358

Reply:
Updated!